src/components/empty/Empty.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+const Empty = () => (
```

Comment:
Are you sure that it should be <table> ? If we are going to you aws desing, it desing look like a table. Could you achieve it with a <div> or <section> ?